server/routes/testutils/requestTestUtils.ts

```diff
@@ -0,0 +1,69 @@
+import { NextFunction, Request, Response } from 'express'
+import { ValidationError } from '../../middleware/validationMiddleware'
+
+const exampleUserDetails = {
```

Comment:
@louisenorris we included this - maybe your cutdown version would be more appropriate?